src/webgl/material.js

```diff
@@ -693,15 +823,63 @@ p5.prototype.emissiveMaterial = function(v1, v2, v3, a) {
  * }
  * function draw() {
  *   background(0);
+ *   let locX = mouseX - width / 2;
+ *   let locY = mouseY - height / 2;
  *   noStroke();
- *   ambientLight(50);
- *   pointLight(250, 250, 250, 100, 100, 30);
+ *   ambientLight(100);
+ *   pointLight(255, 255, 233, locX, locY, 50);
+ *   rotateZ(frameCount * 0.01);
+ *   rotateX(frameCount * 0.01);
+ *   rotateY(frameCount * 0.01);
  *   specularMaterial(250);
- *   sphere(40);
+ *   torus(20, 10, 64, 64);
```

Comment:
let's split this into two examples. one with no rotation or mouse control, and one with both